test/src/server/routes/entity/entity.js

```diff
@@ -80,3 +84,436 @@ describe('getDefaultAliasIndex', () => {
 		expect(getDefaultAliasIndex(aliasSet)).to.equal(2);
 	});
 });
+
+describe('deleteRelationships', () => {
+	const {Author, Work, Revision, WorkRevision, WorkHeader, Relationship, RelationshipSet, RelationshipType, bookshelf} = orm;
+	let author;
+	let editorJSON;
+
+	beforeEach(async () => {
+		const editor = await createEditor();
+		author = await createAuthor();
+		editorJSON = editor.toJSON();
+	});
+	afterEach(truncateEntities);
+	// let author = await createAuthor();s
+	// const workUUID = getRandomUUID();
+	// const authorBbid = author.get('bbid');
+	// await createRelationshipSet(authorBbid, workUUID, 'Work');
+	// author = await author.refresh({withRelated: 'relationshipSet.relationships'});
+	// const authorJSON = author.toJSON();
+	// const authorRelationships = authorJSON.relationshipSet.relationships;
+	it('it should return an empty array if there are no relationships to delete', async () => {
+		const affectedEntities = await deleteRelationships(orm, null, author);
```

Comment:
This test is currently not working properly.
There are two issues:
1. you're passing the author model instead of a JSON object
2. you haven't fetched the relationships before passing it.

With those two things changed, the test still passes even though `createAuthor` creates a relationship. The reason is the same as the test where you have a relationship to a deleted entity: the other entity create by `createAuthor` doesn't have a relationshipSet to modify.